fastcrypto/src/secp256r1/mod.rs

```diff
@@ -299,14 +299,16 @@ impl AsRef<[u8]> for Secp256r1PrivateKey {
 
 impl zeroize::Zeroize for Secp256r1PrivateKey {
     fn zeroize(&mut self) {
+        // Unwrap is safe here because we are using a constant and it has been tested
+        // (see fastcrypto/src/tests/secp256r1_tests::test_sk_zeroization_on_drop)
+        self.privkey = ExternalSecretKey::from_bytes(&Scalar::ONE.to_bytes()).unwrap();
```

Comment:
why is this needed? (was the old comment wrong?)

Reply:
No, I don't think it was. I just added this line to align the implementation with secp256k1.